nuxeo-runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/model/DescriptorRegistry.java

```diff
@@ -61,15 +66,17 @@
                                     .collect(Collectors.toList());
     }
 
-    public boolean register(String target, String xp, Descriptor descpritor) {
+    public boolean register(String target, String xp, Descriptor descriptor) {
+        log.trace("Register {} to {}/{}", descriptor.getId(), target, xp);
```

Comment:
I'd prefer `debug` for all these. We reserve `trace` to logging really low-level and expensive stuff, like all traffic to/from a database or huge outputs.